submarine-server/server-submitter/submitter-yarn/src/main/java/org/apache/submarine/server/submitter/yarn/YarnJobSubmitter.java

```diff
@@ -54,22 +52,18 @@ public void setTonyClient(TonyClient client) {
   @Override
   public ApplicationId submitJob(Parameter parameters)
       throws IOException {
-    if (parameters.getFramework() == Framework.PYTORCH) {
-      // we need to throw an exception, as ParametersHolder's parameters field
-      // could not be casted to TensorFlowRunJobParameters.
-      throw new UnsupportedOperationException(
-          "Support \"–-framework\" option for PyTorch in Tony is coming. " +
-              "Please check the documentation about how to submit a " +
-              "PyTorch job with TonY runtime.");
-    }
-
     LOG.info("Starting Tony runtime..");
 
```

Comment:
Could you add a log entry here about the type of the framework?

Reply:
Done